galeracluster/source/sysdbs.rst

```diff
@@ -0,0 +1,112 @@
+################
```

Comment:
This entire page still refers to `wsrep_schema` instead of `wsrep_*` system pages under `mysq` database.

Reply:
It looks like galeracluster/source/systemtables.rst supersedes this page. If so, this one should be removed.